hexa/pipelines/management/commands/pipelines_runner.py

```diff
@@ -18,6 +19,10 @@
 logger = getLogger(__name__)
 
 
+class PodTerminationReason(Enum):
```

Comment:
It seems that this class is not used - can you confirm?

Reply:
Yes, I've just added a check for the termination reason to see if it was due to timeout and adapt the stdout.